pkgs/by-name/ge/gema/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  installShellFiles,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gema";
+  version = "2.0";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/gema/code.git";
+    rev = "v2-0";
+    hash = "sha256-ZgEw8cIPr5peo2mIz6Vln7JS0uQNI6EctTiw3NaZYN0=";
+  };
+
+  makeFlags = [ "--directory=gema/src" ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 gema/src/gema -t $out/bin
+    installManPage gema/doc/gema.1
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A general purpose text processing utility based on the concept of pattern matching. It reads an input file and copies it to an output file transforming the data as specified by the patterns defined by the user.";
+    homepage = "https://gema.sourceforge.net/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ quag ];
+    platforms = platforms.unix;
+    mainProgram = "gema";
+  };
+}
```

Close the opening parenthesis before `finalAttrs`:
```suggestion
})
```

---

Done. Thanks again for all your advice!